front-end/app/upload_file/page.tsx

```diff
@@ -30,23 +33,53 @@ export default function UploadFile() {
                 });
                 const data = await response.json();
                 setData(data);
-                //We will do something with the data high fidelity version.
-                console.log(data);
                 router.push('/export')
             } catch (error) {
                 console.error('Error uploading file', error);
             }
         }
     };
 
+    const test = () => {
```

Comment:
whats this?

Reply:
Good catch, it was remnants of a previous version